src/reflect/scala/reflect/internal/Flags.scala

```diff
@@ -120,7 +120,8 @@ class ModifierFlags {
   final val ARTIFACT      = 1L << 46      // symbol should be ignored when typechecking; will be marked ACC_SYNTHETIC in bytecode
                                           // to see which symbols are marked as ARTIFACT, see scaladocs for FlagValues.ARTIFACT
   final val DEFAULTMETHOD = 1L << 47      // symbol is a java default method
-  final val ENUM          = 1L << 48      // symbol is an enum
+  final val ENUM          = 1L << 48      // symbol is a java enum
+  final val ANNOTATION    = 1L << 49      // symbol is a java annotation
```

Comment:
i don't fully get what you are saying. in any case, emitting java annotations also needs work in the bytecode generator.